index.html

```diff
@@ -906,69 +906,69 @@ <h3>Anchor button</h3>
             <h3>Buttons with icons</h3>
 
             <button type="button" class="btn btn-primary">
-                <i class="material-icons">ac_unit</i> Iceberg
+                <i class="material-icons">vertical_align_bottom</i> Iceberg
```

Comment:
Completely unrelated, but what about adding aria tags to promote best practices?